lib/src/facts/bsd/filesystem_resolver.cc

```diff
@@ -37,8 +37,10 @@ namespace facter { namespace facts { namespace bsd {
             point.name = fs.f_mntonname;
             point.device = fs.f_mntfromname;
             point.filesystem = fs.f_fstypename;
-            point.size = fs.f_bsize * fs.f_blocks;
-            point.available = fs.f_bsize * fs.f_bfree;
+            point.size = (static_cast<uint64_t>(fs.f_bsize)
```

Comment:
This, I realize, is a stylistic choice. For readability, I've always
limited lines to 80 columns, and the outer parens create the right
alignment on the following line. (It appears that much of the CPP codebase
hews to approx. 80 columns.) Are the parens an annoyance?

On Tue, Jul 5, 2016 at 6:08 PM, Kylo Ginsberg <notifications@github.com>
wrote:

> In lib/src/facts/bsd/filesystem_resolver.cc
> <https://github.com/puppetlabs/facter/pull/1371#discussion_r69663207>:
>
> > @@ -37,8 +37,10 @@ namespace facter { namespace facts { namespace bsd {
> >              point.name = fs.f_mntonname;
> >              point.device = fs.f_mntfromname;
> >              point.filesystem = fs.f_fstypename;
> > -            point.size = fs.f_bsize * fs.f_blocks;
> > -            point.available = fs.f_bsize * fs.f_bfree;
> > +            point.size = (static_cast<uint64_t>(fs.f_bsize)
>
> What are the outer parens for?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/puppetlabs/facter/pull/1371/files/8affc3083f2a5076f659788a387d6c844c967a75#r69663207>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AMVyKjBx4gOu-SxAlIEvvV5RLbQ22Hrcks5qSwAggaJpZM4JFgn3>
> .
>



-- 
*Andy Henroid*
Senior Software Engineer
andy.henroid@puppet.com